index.js

```diff
@@ -131,9 +131,16 @@ module.exports = function(app) {
             }
 
             trace(`starting upload of ${path} to ${params.Bucket}/${params.Key}`);
-
+            const s3_client_local = new S3Client({
+        		credentials: defaultProvider({
+                    clientConfig: {
+                        maxRetries: 3,
+                    },
+                    ignoreCache: true,
+                })
```

Comment:
Could be a diff artifact, but looks like you've got a mix of tabs and spaces here.  Mind double checking?

Reply:
I most likely do - did edit in nano through a ssm session on my raspberry. Will fix an update